src/Components/ResetScrollBar/ResetScrollBar.jsx

```diff
@@ -0,0 +1,11 @@
+import { useEffect } from "react";
+import { useLocation } from "react-router";
+
+export default function ResetScrollbar({ children }) {
+    const location = useLocation();
+    useEffect(() => {
+        window.scrollTo(0, 0);
+    }, [location]);
+
+    return <>{children}</>;
```

Comment:
Re-factoring this line would be better as it would help your code become more readable in my opinion.
```
<ResetScrollBar>{children}</ResetScrollBar>
```

Reply:
I'll do that
Thanks